config/peerpods/podvm/libvirt-podvm-image-handler.sh

```diff
@@ -158,6 +161,185 @@ function download_rhel_kvm_guest_qcow2() {
 
 }
 
+# Function to extract RVPS value from PODVM SE image
+function calculate_rvps_qcow2() {
+    mkdir -p "${CAA_SRC_DIR}"/podvm/rvps_config
+    cp "${CAA_SRC_DIR}"/podvm/output/*.qcow2 "${CAA_SRC_DIR}"/podvm/rvps_config/podvm-libvirt.qcow2
+    export SE_IMG_PATH="${CAA_SRC_DIR}"/podvm/rvps_config/podvm-libvirt.qcow2
+    export RVPS_CONFIG_DIR="${CAA_SRC_DIR}"/podvm/rvps_config
+    rm -rf $RVPS_CONFIG_DIR/se.img
+    rm -rf /mnt/sevm
+    mkdir /mnt/sevm
+    sleep 5
+    modprobe nbd
+
+    if [ $? -eq 0 ]; then
+    echo "modprobe ran successfully"
+    else
+    echo "modprobe command execution Failed!!"
+    fi
+
+    sleep 10
+
+    qemu-nbd -c /dev/nbd3 "${SE_IMG_PATH}" 
+    if [ $? -eq 0 ]; then
+        echo "nbd connection successful.."
+    else
+	    echo "Retrying manual nbd3 creation..."
+	    mknod /dev/nbd3 b 43 96
```

Comment:
Hi @savitrilh , 43 is the major device number for 'nbd' devices (Used for all the nbd's). The second parameter is minor device number which is being decided by +32 starting from nbd1( nbd1 :- 32 , nbd2 :- 64 , nbd3 :- 96 , nbd4 :- 128 etc.). Since we are taking nbd3 so have taken 96 as minor device. Here are some of the references from one of the lpar's for nbd :- 

```
[root@a3elp36 dev]# ls -lrt nbd*
brw-rw----. 1 root disk 43,  64 Jul  8 22:09 nbd2
brw-rw----. 1 root disk 43, 128 Jul  8 22:09 nbd4
brw-rw----. 1 root disk 43, 224 Jul  8 22:09 nbd7
brw-rw----. 1 root disk 43, 192 Jul  8 22:09 nbd6
brw-rw----. 1 root disk 43, 160 Jul  8 22:09 nbd5
brw-rw----. 1 root disk 43, 288 Jul  8 22:09 nbd9
brw-rw----. 1 root disk 43, 256 Jul  8 22:09 nbd8
brw-rw----. 1 root disk 43, 352 Jul  8 22:09 nbd11
brw-rw----. 1 root disk 43, 320 Jul  8 22:09 nbd10
brw-rw----. 1 root disk 43, 416 Jul  8 22:09 nbd13
brw-rw----. 1 root disk 43, 448 Jul  8 22:09 nbd14
brw-rw----. 1 root disk 43, 480 Jul  8 22:09 nbd15
brw-rw----. 1 root disk 43, 384 Jul 11 11:23 nbd12
brw-rw----. 1 root disk 43,  96 Aug 14 08:18 nbd3
```